android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -343,22 +356,21 @@ public void setEventOriginTag(SentryEvent event) {
         if (sdk != null) {
             switch (sdk.getName()) {
                 // If the event is from capacitor js, it gets set there and we do not handle it here.
-                case "sentry.native":
-                    setEventEnvironmentTag(event, "android", "native");
+                case NATIVE_SDK_NAME:
+                    setEventEnvironmentTag(event, "native");
                     break;
-                case "sentry.java.android":
-                    setEventEnvironmentTag(event, "android", "java");
+                case ANDROID_SDK_NAME:
+                    setEventEnvironmentTag(event, "java");
                     break;
                 default:
                     break;
             }
         }
     }
 
-    private void setEventEnvironmentTag(SentryEvent event, String origin, String environment) {
-        event.setTag("event.origin", origin);
-        event.setTag("event.environment", environment);
-    }
+    private void setEventEnvironmentTag(SentryEvent event, String environment) {
+        event.setTag("event.origin", "android");
+        event.setTag("event.environment", environment);    }
```

Comment:
L: Just missing a new line.
```suggestion
        event.setTag("event.environment", environment);
    }
```